src/material-examples/tooltip-custom-class/tooltip-custom-class-example.html

```diff
@@ -0,0 +1,11 @@
+<button mat-raised-button
+        matTooltip="Info about the action"
+        [matTooltipClass]="tooltipClass">
+  Action
+</button>
```

Comment:
For the buttons in these examples, it would be good to add an `aria-label` that explains what is being demonstrated therein.